docker/podman-docker.sh

```diff
@@ -0,0 +1,8 @@
+# DOCKER_HOST initialization
+
+[ -z "$DOCKER_HOST" ] || return
```

Comment:
I don't like this. Try sourcing the file twice (in bash): it will leave you with exit status 1.  

Also, what Nalin said. `id -u` might be safer.

Also, `export foo=bar` is pretty universal, but once upon a time there were shells that did not allow it.  `grep export /etc/profile.d/*.sh` suggests that if those shells still exist, nobody cares about them, and neither do I, but I want to be very explicit about that here.

Also, why the seven-space indentation?

Therefore, suggestion:
```bash
if [ -z "$DOCKER_HOST" ]; then
    if [ `id -u` -eq 0 ]; then
        export DOCKER_HOST=unix:///run/podman/podman.sock
    else
        if [ -n "$XDG_RUNTIME_DIR" ]; then
            export DOCKER_HOST=unix://$XDG_RUNTIME_DIR/podman/podman.sock
        else
            FIXME: do we want a fallback?
        fi
    fi
fi
```

Reply:
I think we just do nothing if XDG_RUNTIME_DIR is not set.